### DISCUSSION TITLE
Adding a way to cuztomize background of home tab and whole browser

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#773] [Ideas]
### What feature would you like to see?

Adding a way to cuztomize background of home tab so that it look good

### Other details

here is a example of a tab that I customize
![image](https://github.com/user-attachments/assets/58816053-08ad-4e1e-8707-e31c4b51d077)

it does cost me a lot of time to figure that out but hope u can add that to the Zen browser ☺️

### The css code i use:
:root {
  --zen-main-browser-background: none !important;
}
html#main-window > body {
  background: url(Omen/Ep8a1_Defiance_Youtube\ Cover_.png) !important;
  background-repeat: no-repeat !important;
  background-size: cover !important;
  background-position: bottom right !important;
  background-attachment: fixed !important;
}
.outer-wrapper {
  background: url(Omen/Ep8a1_Defiance_Youtube\ Cover_.png) !important;
  background-repeat: no-repeat !important;
  background-size: 1366px 768px !important;
  background-position: bottom right !important;
  background-attachment: fixed !important;
}
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/656_

**Comment by Dredragon-Austin** [comment]
@dhruvvk2326 the font I use is Valorant font idk i search it online and i alr provide the code that i have done
